webapp/src/Components/Replay/BasicStats/BasicStatsGrid.tsx

```diff
@@ -9,6 +9,7 @@ import {getReplayBasicStats} from "../../../Requests/Replay"
 import {convertSnakeAndCamelCaseToReadable} from "../../../Utils/String"
 import {StatChart} from "../../Shared/Charts/StatChart"
 import {LoadableWrapper} from "../../Shared/LoadableWrapper"
+import {faRocket} from "@fortawesome/free-solid-svg-icons/faRocket"
```

Comment:
My only comment would be that I think all our imports for these are {faIcon} from "@fortawesome/free-solid-svg-icons" (ie without the "/faIcon" at the end) but it really doesn't matter.